utilities/verify_single_it.sh

```diff
@@ -21,7 +21,7 @@ echo "----- building and installing parent pom -----"
 mvn -B -N install -DskipTests
 
 echo "----- building and installing shared modules -----"
-mvn -B -pl google-cloud-core,google-cloud-core-http,google-cloud-core-grpc install -DskipTests
+mvn -B -pl google-cloud-core,google-cloud-core-http,google-cloud-core-grpc,google-cloud-storage,google-cloud-pubsub install -DskipTests
```

Comment:
why did storage and pubsub get put in this command?

Reply:
It was the easiest way to deal with the fact that bigquery depends on storage, and notification depends on pubsub (and storage).